apply/cloud_applier.go

```diff
@@ -43,6 +45,7 @@ func NewAliCloudProvider(cluster *v1.Cluster) Interface {
 		ImageManager:   image.NewImageService(),
 		FileSystem:     filesystem.NewFilesystem(),
 		Guest:          guest.NewGuestManager(),
+		Config:         config.NewConfiguration(cluster.Name),
```

Comment:
cloud mod using config is a litter complexity，need copy the Clusterfile configs to the remote server, I will do it later, delete this now.

Reply:
OK